components/MenuModal.js

```diff
@@ -25,23 +37,30 @@ function MenuModal() {
           <span className="hamburger-inner" />
         </span>
       </Button>
-      <StyledModal size="lg" aria-labelledby="title" show={show} onHide={close}>
+      <StyledModal
```

Comment:
Let's alphabetize the props. :)

Reply:
Done in https://github.com/aragon/anj.aragon.org/pull/23/commits/1002e09f752f58c3d7bf1898f54a89c8a403bfa1